docs/HowToUseRamdump.md

```diff
@@ -0,0 +1,76 @@
+# How to Use Ramdump
+
+## Contents
+> [Ramdump Upload Steps](#ramdump-upload-steps)  
+> [Ramdump Parsing Steps](#ramdump-parsing-steps)  
+
+## Ramdump Upload Steps
+Below configuration must be enabled to support ramdump upload  
+1. Enable crashdump support (CONFIG_BOARD_CRASHDUMP=y)
+```
+Hardware Configuration > Board Selection -> Enable Board level logging of crash dumps to y
+```
+2. Enable ramdump (CONFIG_BOARD_RAMDUMP_UART=y)
+```
+Hardware Configuration > Board Selection -> Enable ramdump via UART to y
+```
+3. Enable Stack Dump (CONFIG_ARCH_STACKDUMP=y)
+```
+Hardware Configuration > Chip Selection -> Dump stack on assertions to y
+```
+4. Enable Debug (CONFIG_DEBUG=y)
+```
+Debug Options -> Enable Debug Output Features to y
+```
+5. Enable frame pointer for generating stack frame (CONFIG_FRAME_POINTER=y)
+```
+Debug Options -> Enable backtracking using Frame pointer register  to y
+```
+
+Run ramdump tool from host after target crashed.  
+1. If target got PANIC (asserted), below messages will be promted.
+```
+	****************************************************
+	Disconnect this serial terminal and Run Ramdump Tool
+	****************************************************
+```
+2. Disconnect/close serial termial (may be minicom)  
+
+3. Run ramdump tool
+```
+cd tools/ramdump/
+sh ramdump.sh
+```
+4. Ramdump Tool recieves the ram contents from target.
+```
+Target Handshake successful do_handshake
+Target entered to ramdump mode
+Receiving ramdump......
+ramdump_recv: ramdump_address = 02020000, ramdump_size = 968704
+[===========================================================>]
+Ramdump received successfully
+copying ramdump_0x02020000_0x0210c800.bin to  /xyz-path
+```
+
+## Ramdump Parsing Steps
+1. Run Ramdump Parser Script
+```
+cd ramdump_parser/
```

Comment:
There is no ramdump_parser folder.

Reply:
@sunghan-chang , I will create a pull request with ramdump_parser folder and stripped version of the  callTrace.py in sometime, Kindly keep this PR open until then.

Reply:
@sunghan-chang , I raised a PR#1015 for python script (callTrace.py) at new folder tools/ramdump_parser/callTrace.py  [ https://github.com/Samsung/TizenRT/pull/1015 ]

Reply:
I left a comment there.

Reply:
we will modifying accordingly.